torch/nn/modules/module.py

```diff
@@ -91,8 +91,12 @@ def register_buffer(self, name, tensor):
         """
         if hasattr(self, name) and name not in self._buffers:
             raise KeyError("attribute '{}' already exists".format(name))
-
-        self._buffers[name] = tensor
+        elif tensor is not None and not torch.is_tensor(tensor):
```

Comment:
I think `torch.is_tensor` is deprecated in favor of `isinstance(x, torch.Tensor)`. Is that right @colesbury?

Reply:
I think yes https://github.com/pytorch/pytorch/pull/5473

Reply:
I don't see the deprecation message when testing. Should I add it in `torch.is_tensor`?